src/main/java/games/strategy/engine/data/GameDataMemento.java

```diff
@@ -0,0 +1,168 @@
+
+/**
+ * Provides factory methods for creating objects that can import mementos to and export mementos from game data.
+ */
+public final class GameDataMemento {
+  @VisibleForTesting
+  static final long CURRENT_SCHEMA_VERSION = 1L;
+
+  @VisibleForTesting
+  static final String SCHEMA_ID = "application/x.triplea.game-data-memento";
+
+  /**
+   * An immutable collection of default memento export options.
+   *
+   * <p>
+   * The key is the option name. The value is the option value.
+   * </p>
+   */
+  public static final Map<ExportOptionName, Object> DEFAULT_EXPORT_OPTIONS_BY_NAME = newDefaultExportOptionsByName();
+
+  private GameDataMemento() {}
+
+  @VisibleForTesting
+  static final class PropertyNames {
+    static final String NAME = "name";
+    static final String VERSION = "version";
+
+    private PropertyNames() {}
+  }
+
+  private static Map<ExportOptionName, Object> newDefaultExportOptionsByName() {
+    return Maps.immutableEnumMap(ImmutableMap.of(
+        ExportOptionName.EXCLUDE_DELEGATES, false));
+  }
+
+  /**
+   * Creates a new game data memento exporter using the default export options.
+   *
+   * @return A new game data memento exporter; never {@code null}.
+   */
+  public static MementoExporter<GameData> newExporter() {
+    return newExporter(DEFAULT_EXPORT_OPTIONS_BY_NAME);
+  }
+
+  /**
+   * Creates a new game data memento exporter using the specified export options.
+   *
+   * @param optionsByName The memento export options; must not be {@code null}. The key is the option name. The value is
+   *        the option value.
+   *
+   * @return A new game data memento exporter; never {@code null}.
+   */
+  public static MementoExporter<GameData> newExporter(final Map<ExportOptionName, Object> optionsByName) {
+    checkNotNull(optionsByName);
+
+    return new PropertyBagMementoExporter<>(SCHEMA_ID, CURRENT_SCHEMA_VERSION, new ExportHandlers(optionsByName));
+  }
+
+  /**
+   * Names the options that can be specified when creating a new game data memento exporter.
+   */
+  public static enum ExportOptionName {
+    /**
+     * Indicates delegates should be excluded from the memento (value type: {@code Boolean}).
+     */
+    EXCLUDE_DELEGATES
+  }
+
+  private static final class ExportHandlers implements PropertyBagMementoExporter.HandlerSupplier<GameData> {
+    private final Map<Long, PropertyBagMementoExporter.Handler<GameData>> handlersBySchemaVersion =
+        getHandlersBySchemaVersion();
+
+    // TODO: add support for options
+    @SuppressWarnings("unused")
+    private final Map<ExportOptionName, Object> optionsByName;
+
+    ExportHandlers(final Map<ExportOptionName, Object> optionsByName) {
+      this.optionsByName = new HashMap<>(optionsByName);
+    }
+
+    private Map<Long, PropertyBagMementoExporter.Handler<GameData>> getHandlersBySchemaVersion() {
+      return ImmutableMap.<Long, PropertyBagMementoExporter.Handler<GameData>>builder()
+          .put(1L, this::exportPropertiesV1)
```

Comment:
It might be that it is late, though the 1L is looking a bit like a magic number. Is this where we tie `exportPropertiesV1` to the value `1L`? I am a bit confused.. Even though it may be easier to remove version down the road, I have been thinking it may make sense to do it now:
- simpler code now, not later
- we unify to our final version faster, perhaps there are surprises when using single version. Regardless we would be able to review/finalize our usage of versioning sooner
- easier review